backend/sokdak/src/main/java/com/wooteco/sokdak/member/service/EmailSenderImpl.java

```diff
@@ -16,6 +17,7 @@ public EmailSenderImpl(MailProperties properties, JavaMailSender javaMailSender)
         this.javaMailSender = javaMailSender;
     }
 
+    @Async
```

Comment:
```suggestion
    @Async("mailExecutor")
```
현재는 executor를 하나만 등록해 주었기 때문에 등록한 executor로 돌아가지만, 다른 executor가 생겼을 때는 SimpleAsyncTaskExecutor 로 돌아간다고 하네요! 다음에 혹시나 다른 executor를 추가할 때 혼선이 올 것 같아 executor name을 적어주는게 좋을 것 같아요!